partner_communication/views/ir_attachment_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Attachment -->
+        <record id="view_ir_attachment_form" model="ir.ui.view">
+            <field name="model">ir.attachment</field>
+            <field name="inherit_id" ref="base.view_attachment_form"/>
```

Comment:
This is already done in file `communication_job_view.xml`, line 117. Please move the record in that file inside your file (it will override your content).